gammapy/maps/tests/test_base.py

```diff
@@ -50,6 +53,31 @@ def test_map_from_geom():
     assert isinstance(m, HpxNDMap)
 
 
+@pytest.mark.parametrize(('binsz', 'width', 'map_type', 'skydir', 'axes', 'unit'),
+                         mapbase_args_with_axes)
+def test_map_get_image_by_coord(binsz, width, map_type, skydir, axes, unit):
+    m = Map.create(binsz=binsz, width=width, map_type=map_type,
+                   skydir=skydir, axes=axes, unit=unit)
+    coords = (1.234,) * len(m.geom.axes)
+    m_im_tuple = m.get_image_by_coord(coords)
+
+    coords_dict = {}
+    for value, axes in zip(coords, m.geom.axes):
+        coords_dict[axes.name] = value * Unit(axes.unit)
+
+    m_im_dict = m.get_image_by_coord(coords_dict)
+    assert_equal(m_im_tuple.data, m_im_dict.data)
+
+
+@pytest.mark.parametrize(('binsz', 'width', 'map_type', 'skydir', 'axes', 'unit'),
+                         mapbase_args_with_axes)
+def test_map_get_image_by_pix(binsz, width, map_type, skydir, axes, unit):
+    m = Map.create(binsz=binsz, width=width, map_type=map_type,
+                   skydir=skydir, axes=axes, unit=unit)
+    pix = (0.1234,) * len(m.geom.axes)
+    m_im = m.get_image_by_pix(pix)
+
```

Comment:
Is it worth adding a `test_map_get_image_by_idx`? Or the examples from the docstring?